### ISSUE TITLE
Changable Texture Filtering and Mipmapping on SpriteFonts

### ISSUE THREAD

**Issue by scrapSavage** [Yellow-Dog-Man/Resonite-Issues#2267]
### Is your feature request related to a problem? Please describe.

Currently there are no ways to turn off or edit mipmapping on sprite fonts, leading to artifacting when viewed at larger distances, and there is no way to change texture filtering on spritefonts making sprites bleed into other sprites

### Describe the solution you'd like

Add a texture filter parameter and a mipmaps checkbox

### Describe alternatives you've considered

Dealing with it

### Requesters

scrapSavage
ShadowX

**Comment by Frooxius** [comment]
Can you elaborate what do you mean? The StaticFont component has MipMaps and MipMapFiltering options.

**Comment by scrapSavage** [comment]
Yes, the StaticFont component does, but the DynamicSpriteFont component does not.

**Comment by shiftyscales** [comment]
Updated the issue to be more specific to to the fact you are referring to DynamicSpriteFont, and added an accompanying image to visually display the request, @scrapSavage.

It is best to be as precise as possible when making reports, making use of exact component names, etc. Likewise- having visual references, e.g. images/video can be useful in further displaying what your issues are.

This issue has everything it needs to be considered now however, @Frooxius @ProbablePrime.
